docs_source_files/content/support_packages/working_with_colours.en.md

```diff
@@ -147,8 +147,8 @@ and converted into a valid Color object:
 
 {{< code-tab >}}
   {{< code-panel language="java" >}}
-Color loginButtonColour = driver.findElement(By.id("login")).getCssValue("color");
-Color loginButtonBackgroundColour = driver.findElement(By.id("login")).getCssValue("background-color");
+Color loginButtonColour = Color.fromString(driver.findElement(By.id("login")).getCssValue("color"));
```

Comment:
```suggestion
String loginButtonColour = driver.findElement(By.id("login")).getCssValue("color");
```
I have checked the code sample with selenium alpha4, This will the fix the above. I think there is no need to use `staticFunction` to get the color

Also,
String loginButtonColour = driver.findElement(By.id("login")).getCssValue("color");
(and)
Color loginButtonColour = Color.fromString(driver.findElement(By.id("login")).getCssValue("color"));

return the same output.

Reply:
no, these aren't same. `getCssValue` gives you the computed `rgba` as string whereas using the static function returns `Color` object that can be compared to any other Color object. This is useful when you need to compare two objects derived using different mechanism, such as hash value vs rgba. You can refer to the documentation. I advised a fix because, the documentation says in the next line of getting the color `assert loginButtonBackgroundColour.equals(HOTPINK);`, this can be possible only when you've a valid Color object which `getCssValue` doesn't return. Does it makes sense?